### ISSUE TITLE
Ctrlr x64 v5.5.9: cannot get properties via Lua anymore

### ISSUE THREAD

**Issue by Possemo** [RomanKubiak/ctrlr#146]
Dear Roman. I built v5.5.9 as the provided prebuilt v5.6.0 does not work at all for me - v5.6.0 throws an error when I select a MIDI-Port. Furthermore, v5.6.0 just quits without an error when I run the Lua scripts with the "getProperty" code as described here. I suspect that this is the same problem, it just crashes badly instead of throwing a Lua-error.

```
presetSelector=function(modulator,value)

    test = panel:getModulatorByName("PresetSel1"):getProperty("modulatorCustomName") 

end
```

> Error message: [string "presetSelector"]:3: attempt to call method 'getProperty' (a nil value)

My panels rely on the ability to get properties like "modulatorCustomIndex", "modulatorCustromName", modulatorCustomIndexGroup" etc. As far as I could test "getProperty" or "getPropertyInt" and "getPropertyString" does not work for all properties.

**Comment by rkubiak01** [comment]
i'll check that asap

**Comment by rkubiak01** [comment]
i found the bug, i'll fix that in the next release

**Comment by Possemo** [comment]
fine, thanks - looking forward for the next release.

**Comment by Possemo** [comment]
Ah, I now saw that this  is already fixed in the newest code. It is probably "unknown" that has fixed it: 
![fix](https://user-images.githubusercontent.com/21274845/107014350-c48b7e00-679b-11eb-8761-ee89478f2cae.jpg)
I had a hard time building the latest code, now I finally succeded - and I saw that the bug has gone away.

But, in the newest code, windows x64 there is another big bug. I cannot select MIDI-Ports. I will open a new Issue for that.
